.github/workflows/prepend-license.yml

```diff
@@ -0,0 +1,35 @@
+on:
+  push:
+    branches:
+      - develop
+      - 'release/**'
+
+name: Prepend licences to files
+jobs:
+  prepend_license: 
+    name: Commit prepended licences to files 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GH_ADMIN }}
```

Comment:
needs the PAT of an admin to bypass the protected branch.

Reply:
Could this be dangerous in that anyone can modify the script and run any git command on develop?